rskj-core/src/test/java/co/rsk/vm/VmDslTest.java

```diff
@@ -118,53 +125,189 @@ void testPush0() throws FileNotFoundException, DslProcessorException {
-        Assertions.assertNotNull(statusCreationNew);
+        assertNotNull(statusCreationNew);
         Assertions.assertEquals(1, statusCreationNew.length);
         Assertions.assertEquals(1, statusCreationNew[0]);
 
         Transaction callTransactionNew = world.getTransactionByName("txCallNew");
-        Assertions.assertNotNull(callTransactionNew);
+        assertNotNull(callTransactionNew);
         TransactionReceipt callTransactionReceiptNew = world.getTransactionReceiptByName("txCallNew");
-        Assertions.assertNotNull(callTransactionReceiptNew);
+        assertNotNull(callTransactionReceiptNew);
         byte[] statusCallNew = callTransactionReceiptNew.getStatus();
-        Assertions.assertNotNull(statusCallNew);
+        assertNotNull(statusCallNew);
         Assertions.assertEquals(1, statusCallNew.length);
         Assertions.assertEquals(1, statusCallNew[0]);
 
         short newGas = ByteBuffer.wrap(callTransactionReceiptNew.getGasUsed()).getShort();
 
         Block block4 = world.getBlockByName("b04");
-        Assertions.assertNotNull(block4);
+        assertNotNull(block4);
         Assertions.assertEquals(1, block4.getTransactionsList().size());
 
 
         Transaction creationTransactionOld = world.getTransactionByName("txCreateOld");
-        Assertions.assertNotNull(creationTransactionOld);
+        assertNotNull(creationTransactionOld);
         TransactionReceipt creationTransactionReceiptOld = world.getTransactionReceiptByName("txCreateOld");
-        Assertions.assertNotNull(creationTransactionReceiptOld);
+        assertNotNull(creationTransactionReceiptOld);
         byte[] statusCreationOld = creationTransactionReceiptNew.getStatus();
-        Assertions.assertNotNull(statusCreationOld);
+        assertNotNull(statusCreationOld);
         Assertions.assertEquals(1, statusCreationOld.length);
         Assertions.assertEquals(1, statusCreationOld[0]);
 
         Transaction callTransactionOld = world.getTransactionByName("txCallOld");
-        Assertions.assertNotNull(callTransactionOld);
+        assertNotNull(callTransactionOld);
         TransactionReceipt callTransactionReceiptOld = world.getTransactionReceiptByName("txCallOld");
-        Assertions.assertNotNull(callTransactionReceiptOld);
+        assertNotNull(callTransactionReceiptOld);
         byte[] statusCallOld = callTransactionReceiptOld.getStatus();
-        Assertions.assertNotNull(statusCallOld);
+        assertNotNull(statusCallOld);
         Assertions.assertEquals(1, statusCallOld.length);
         Assertions.assertEquals(1, statusCallOld[0]);
 
         short oldGas = ByteBuffer.wrap(callTransactionReceiptOld.getGasUsed()).getShort();
-        Assertions.assertTrue(newGas < oldGas);
+        assertTrue(newGas < oldGas);
+    }
+
+    @Test
+    void testInitCodeSizeValidationSuccessWithoutInitcodeCostViaCreateOpcodeCREATE() throws FileNotFoundException, DslProcessorException {
```

Comment:
I personally try to match the dsl `.txt` files with the test name to avoid misunderstandings. I think it's clearer that way, but if it's too much work, forget about it.

Reply:
Ok, will do.